src/templates/fr/2019/accessibility_statement.html

```diff
@@ -0,0 +1,123 @@
+
+        <p>
+          Le Web Almanac prend les mesures suivantes pour assurer l’accessibilité de ce site Web&nbsp;:
+        </p>
+        <ul>
+          <li>Inclure l’accessibilité dans le cadre du développement de ce site Web.</li>
+          <li>Vérifier l’accessibilité à l’aide des outils disponibles.</li>
+          <li>Être transparent sur les problèmes d’accessibilité <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues?utf8=%E2%9C%93&q=label%3AAccessibility+" hreflang="en">sur notre dépôt Github</a>.</li>
+          <li>Résoudre les problèmes d’accessibilité soulevés en temps opportun.</li>
+          <li>Publier et tenir à jour cette déclaration d’accessibilité.</li>
+        </ul>
+      </section>
+      
+      <section>
+        <h2 id="statut-conformite"><a href="#statut-conformite" class="anchor-link">Statut de conformité</a></h2>
+
+        <p>
+          Les <a href="https://www.w3.org/WAI/standards-guidelines/wcag/" hreflang="en">directives pour l’accessibilité du contenu Web (WCAG)</a> définissent les exigences pour les concepteurs et les développeurs afin d’améliorer l’accessibilité pour les personnes handicapées. Il définit trois niveaux de conformité&nbsp;: niveau A, niveau AA et niveau AAA. Le Web Almanac est partiellement conforme au niveau AAA des <a href="https://www.w3.org/TR/WCAG21/">WCAG 2.1</a>. Partiellement conforme signifie que certaines parties du contenu ne sont pas entièrement conformes à la norme d’accessibilité.
+        </p>
+
+        <p>
+          Nous nous considérons partiellement conformes parce que nous n’avons pas effectué un audit complet et vérifié de l’extérieur du site Web, ni que le site Web n’a été examiné par les utilisateurs de lecteurs d’écran, nous ne pouvons donc pas être sûrs de notre état actuel. Nous avons audité toutes les pages du site Web à l’aide d’un certain nombre d’outils, notamment <a href="https://wave.webaim.org/" hreflang="en">l’outil d’accessibilité WAVE</a>, <a href="https://hacks.mozilla.org/2019/10/auditing-for-accessibility-problems-with-firefox-developer-tools/" hreflang="en">l’inspecteur d’accessibilité Firefox</a> et le <a href="https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd" rel="nofollow">plugin AX Chrome</a>.
+        </p>
+
+        <p>
+          Le but de l’Almanach Web est de fournir des informations détaillées sur l’état du Web. Bien que nous cherchions à nous assurer que tout le contenu est facilement lisible, même par des non-experts, la nature du matériel peut signifier qu’il peut y avoir un niveau d’expertise et de compréhension technologique nécessaire pour comprendre certains chapitres et certaines mesures, qui peuvent ne pas répondre au <a href="https://www.w3.org/WAI/WCAG21/quickref/#reading-level" hreflang="en">niveau WCAG 2.1. des directives de niveau de lecture AAA</a>.
```

Comment:
Typo "Almanach"
+ Inclusivité

```suggestion
          Le but de l’Almanac Web est de fournir des informations détaillées sur l’état du Web. Bien que nous cherchions à nous assurer que tout le contenu est facilement lisible, même par des personnes non-expertes, la nature du matériel peut signifier qu’il peut y avoir un niveau d’expertise et de compréhension technologique nécessaire pour comprendre certains chapitres et certaines mesures, qui peuvent ne pas répondre au <a href="https://www.w3.org/WAI/WCAG21/quickref/#reading-level" hreflang="en">niveau WCAG 2.1. des directives de niveau de lecture AAA</a>.
```